mlflow/models/resources.py

```diff
@@ -132,21 +140,16 @@ class DatabricksVectorSearchIndex(DatabricksResource):
     Args:
         index_name (str): The name of all the databricks vector search index names
         used by the model.
```

Comment:
```suggestion
        index_name (str): The name of the databricks vector search index
        used by the model.
```

I guess the original desscription is incorrect? This represent a single index not a list of them.